tests/sentry/seer/anomaly_detection/test_store_data.py

```diff
@@ -70,13 +74,64 @@ def test_anomaly_detection_format_historical_data_two(self):
             {"time": self.time_1_ts},
             {"count_unique_tags_sentry_user": 1, "time": self.time_2_ts},
         ]
-        data = SnubaTSResult({"data": snuba_raw_data}, self.time_1_ts, self.time_2_ts, 3600)
-        result = format_historical_data(data, errors)
+        data = SnubaTSResult(
+            data={"data": snuba_raw_data}, start=self.time_1_dt, end=self.time_2_dt, rollup=3600
+        )
+        result = format_historical_data(
+            data=data,
+            query_columns=["count_unique_tags_sentry_user"],
+            dataset=errors,
+            organization=self.organization,
+        )
         assert result == expected_return_value
 
     def test_anomaly_detection_fetch_historical_data(self):
         alert_rule = self.create_alert_rule(organization=self.organization, projects=[self.project])
         snuba_query = SnubaQuery.objects.get(id=alert_rule.snuba_query_id)
+        # CEO: despite passing event_type to store_event, the event type is default
```

Comment:
I need to figure this out, it's causing issues in `test_organization_alert_rule_anomalies.py`

Reply:
Something changed in Relay because after normalization the event type is changed to "default" https://github.com/getsentry/sentry/blob/master/src/sentry/event_manager.py#L434-L435 

I had added `event_type` to `self.store_event` and it used to work, I'm surprised tests haven't broken because of this.